pkgs/development/libraries/gpgme/default.nix

```diff
@@ -51,12 +50,16 @@ stdenv.mkDerivation rec {
     pkg-config
     texinfo
   ] ++ lib.optionals pythonSupport [
+    buildPackages.python3
```

Comment:
That will be the wrong version in `python311Packages.gpgme`.

Reply:
Use `python3.pythonForBuild` instead.

Reply:
c88f4b86cbdc